### PR TITLE
[TTT] Setup deployment for Production

### PR DESCRIPTION
**Pull request by Alonski**
This sets up TTT to be ready for deployment from the `deploy` branch.

### CODE REVIEW

File: packages/tic-tac-toe/config/dotenv.js [hunk 1]
```diff
@@ -6,6 +6,19 @@ const path = require('path');
 const fs = require('fs');
 
 function configureEnvVariables(env) {
+  // State Channel Environment
+  // Intended usage is a single file in monorepo root defining configuration for multiple packages
+  const SC_ENV = process.env.SC_ENV;
```
Comment (geoknee) [inline packages/tic-tac-toe/config/dotenv.js:11]: Any reason not to import this from @statechannels/devtools?

File: packages/tic-tac-toe/config/env.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ const REACT_APP = /^REACT_APP_/i;
 // These are Magmo specific values that should be injected into the application
 const MAGMO_VALUES = [
```
Comment (geoknee) [inline packages/tic-tac-toe/config/env.js:33]: This seems like an out-of-date name

Comment (Alonski) [reply packages/tic-tac-toe/config/env.js:33]: I copy pasted this from somewhere and I don't remember where :)
I don't even think I need this file anymore hmm. :)

File: packages/tic-tac-toe/config/dotenv.js (hunk 1 above)
Comment (Alonski) [reply packages/tic-tac-toe/config/dotenv.js:11]: `devtools` requires the env vars directly in this function.
In Ember I use an Ember Addon to inject the env vars during runtime.
If the devtools function was split up into two functions (one that returns a path and one that requires the env vars) I could utilize it.

**Pull request merged**